addons/mail/static/src/core/common/message_model.js

```diff
@@ -41,6 +44,11 @@ export class Message extends Record {
     author = Record.one("Persona");
     body = Record.attr("", { html: true });
     composer = Record.one("Composer", { inverse: "message", onDelete: (r) => r.delete() });
+    linkCopyable = Record.attr(false, {
+        compute() {
+            return this.message_type;
+        },
+    });
```

Comment:
I don't understand this compute, the message type could be directly used instead of this?

Reply:
It's initially it was to make it patchable in other modules where thy might not need this. But, I think we can use this directly.

Reply:
I guess we can worry about patching this when we actually need the patch :)